src/main/kotlin/fi/metatavu/timebank/api/forecast/ForecastService.kt

```diff
@@ -52,4 +52,24 @@ class ForecastService {
         return doRequest("/v2/persons")
     }
 
+    /**
+     * Gets time registrations from Forecast
+     *
+     * @OptionalParam date after in YYYY-MM-DD LocalDate
+     * @return Response with time registrations data
+     */
+    fun getTimeEntries(after: LocalDate?): String? {
+        var path = "/v3/time_registrations"
```

Comment:
V3 is marked as deprecated (https://github.com/Forecast-it/API/blob/master/sections/time_registrations.md#time-registrations). Use V4 instead